lib/helpers/frontend/handle-raw-image.ts

```diff
@@ -0,0 +1,14 @@
+import { SetStateAction } from "react";
+
+export const handleRawImage = (
+  image: Blob,
+  setImageUrl: (value: SetStateAction<string | null>) => void
```

Comment:
This should just be `(imageUrlString: string) => void`. Something is weird here.

Reply:
Also, the convention would typically be to just call this parameter `callback` for this pattern